game-app/game-core/src/test/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcherTest.java

```diff
@@ -1,15 +1,119 @@
 package games.strategy.engine.framework.startup.ui;
 
+import static games.strategy.engine.framework.CliProperties.LOBBY_GAME_COMMENTS;
 import static games.strategy.engine.framework.startup.ui.InGameLobbyWatcher.getLobbySystemProperty;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.nullValue;
-
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import games.strategy.engine.framework.startup.WatcherThreadMessaging;
+import games.strategy.net.INode;
+import games.strategy.net.IServerMessenger;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Optional;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.triplea.http.client.lobby.game.lobby.watcher.GamePostingRequest;
+import org.triplea.http.client.lobby.game.lobby.watcher.GamePostingResponse;
+import org.triplea.http.client.web.socket.client.connections.GameToLobbyConnection;
 
 final class InGameLobbyWatcherTest {
+
+  @Test
+  public void testNewInGameLobbyWatcher_hostNotReachable() throws UnknownHostException {
```

Comment:
We have a convention to just throw `Exception` on test cases, so we don't have to bother with the specifics of the exception details.

Reply:
updated these two tests to just throw base Exception now